src/plugins/data/server/search/search_service.ts

```diff
@@ -105,13 +112,26 @@ export class SearchService implements Plugin<ISearchSetup, ISearchStart> {
       registerUsageCollector(usageCollection, this.initializerContext);
     }
 
+    const aggs = this.aggsService.setup({ registerFunction });
+
+    this.initializerContext.config
```

Comment:
Could we pass config in instead of the whole initializer context?

Reply:
I think the general pattern for plugins/services is to pass in the entire initializer context in the constructor. (I didn't change the signature here to add an additional parameter, just specified what the config schema should look like. I did, however, change the signature on the public service, which matches the autocomplete service.)